vivisect/reports/undeftargets.py

```diff
@@ -18,15 +18,15 @@ def report(vw):
             if seg is not None:
                 sname = seg[v_const.SEG_NAME]
             try:
-                b = binascii.hexlify(vw.readMemory(tova, 8))
+                b = binascii.hexlify(vw.readMemory(tova, 8)).decode()
```

Comment:
probably should move this routine ("encode as hex") into a utility function.

also you know what im going to say about the encoding